yearn/partners/partners.py

```diff
@@ -272,4 +272,14 @@
                 ),
             ],
         ),
+    Partner(
+      name='shapeshiftdao',
+      treasury='0x90a48d5cf7343b08da12e067680b4c6dbfe551be',
+      wrappers=[
+          WildcardWrapper(
+              name='ShapeShift DAO Router',
+              wrapper='0x6a1e73f12018d8e5f966ce794aa2921941feb17e',
```

Comment:
Not sure if it matters, but we since the other addresses in this file are the checksum addresses, we should use the checksums for ours as well.

Addresses do look correct to me other than that.

Reply:
done